### DISCUSSION TITLE
Is it possible to leave pdf comments?

### DISCUSSION THREAD

**Discussion by N-Demir** [ahrm/sioyek#1254] [Ideas]
Searched through the repo but didn't see anything. Is there anyway to leave pdf comments similar to acrobat reader? I find that very useful when reading and highlighting papers. Thanks!

**Comment by ahrm** [comment]
If you mean something like freetext bookmarks then yes, it is added in the development version (it was not possible in the 2.0 release).

**Reply by N-Demir** [reply]
thanks!
